sdcflows/cli/tests/test_find_estimators.py

```diff
@@ -133,17 +133,21 @@
 )
 def test_cli_finder_wrapper(tmp_path, capsys, test_id, config, estimator_id):
     """Test the CLI with --dry-run."""
+    from nibabel.filebasedimages import ImageFileError
+
     import sdcflows.config as sc
 
     # Reload is necessary to clean-up the layout config between parameterized runs
     reload(sc)
 
     path = (tmp_path / test_id).absolute()
     generate_bids_skeleton(path, config)
-    with pytest.raises(SystemExit) as wrapped_exit:
+    # This was set to raise a SystemExit, but was only raising an ImageFileError
+    with pytest.raises(ImageFileError) as wrapped_exit:
```

Comment:
This was probably expecting valid headers in the images.

Reply:
I reverted the changes I made to this test since other PRs don't seem to have an issue. I'm not sure why the problem is happening in this PR only, but I've tried to remove any extraneous changes in case those were causing the problem.